### PR TITLE
Changed data lock clean up scripts to match those from DC

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

**Review by chrisfoster76 — APPROVED** [review]

File: src/SharedPipelineComponents/Datalock/DeployPeriodEnd/sql/ddl/PeriodEnd.Deds.DataLock.DDL.sprocs.sql [hunk 1]
```diff
@@ -11,4 +11,31 @@ IF EXISTS(select 1 from sys.procedures where name = 'CleanupDeds')
 BEGIN
     DROP PROCEDURE DataLock.CleanupDeds
 END
+GO
```
Comment (carlbm) [inline src/SharedPipelineComponents/Datalock/DeployPeriodEnd/sql/ddl/PeriodEnd.Deds.DataLock.DDL.sprocs.sql:14]: This seems like a change in functionality. Losing the current period filters?

**Review by carlbm — CHANGES_REQUESTED** [review]

File: src/SharedPipelineComponents/Datalock/DeployPeriodEnd/sql/ddl/PeriodEnd.Deds.DataLock.DDL.sprocs.sql (hunk 1 above)
Comment (chrisfoster76) [reply src/SharedPipelineComponents/Datalock/DeployPeriodEnd/sql/ddl/PeriodEnd.Deds.DataLock.DDL.sprocs.sql:14]: What current period filters do you mean?

Comment (chrisfoster76) [reply src/SharedPipelineComponents/Datalock/DeployPeriodEnd/sql/ddl/PeriodEnd.Deds.DataLock.DDL.sprocs.sql:14]: Oh, I see them. @MichaelYoung1981 can you address?

Comment (MichaelYoung1981) [reply src/SharedPipelineComponents/Datalock/DeployPeriodEnd/sql/ddl/PeriodEnd.Deds.DataLock.DDL.sprocs.sql:14]: Gary has confirmed I was wrong!  I've changed it to as per production and will create a story to remove linked server delete.

**Review by carlbm — APPROVED** [review]

**Comment by satya-sanikommu** [comment]
Testing completed. This is the change made by DC in production to delete any existing data locks on DEDS prior to copying the new ones, which requires merging into our codebase to be included in the future releases. No functional changes and no regression issues. Ready to merge

**Pull request merged**